src/components/AddBook.js

```diff
@@ -32,12 +32,14 @@ export default function AddBook() {
 
   return (
     <form className="add-new-book" onSubmit={(e) => submitBookToStore(e)}>
-      <h2 className="title">ADD NEW BOOK</h2>
-      <input type="text" placeholder="Add Book" onChange={(e) => setTitle(e.target.value)} />
-      <select name="cars" id="cars">
-        <option value="Under Construction">Categories</option>
-      </select>
-      <input type="submit" value="ADD BOOK" />
+      <div className="add-form horizontal-line-70">
+        <h2 className="title-form">ADD NEW BOOK</h2>
+        <input className="input-one" type="text" placeholder="Add Book" onChange={(e) => setTitle(e.target.value)} />
+        <select name="cars" id="cars">
+          <option value="Under Construction">Categories</option>
+        </select>
+        <input className="inout-two" type="submit" value="ADD BOOK" />
+      </div>
     </form>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/155747892-0e0b53d6-8c91-4aa5-a21f-d3ed68d72ce2.png)

You can solve this linters by adding the `key` property on these stages :point_up: 

### before adding key attributes.
![image](https://user-images.githubusercontent.com/65068771/155748326-79797032-e2d4-4bc4-9757-e32593f05b75.png)

### example after adding them

![image](https://user-images.githubusercontent.com/65068771/155748512-0c34aada-fe24-41e3-9f76-866e4c0660ea.png)

Reply:
Done